elements/barcode-reader/barcode-reader.js

```diff
@@ -31,10 +31,10 @@ class BarcodeReader extends LitElement {
         border-color: rgba(0, 0, 0, 0.5);
         border-width: 5px;
       }
-      #hidden {
+      .hidden {
```

Comment:
`:host([hidden]) { display: none; }` in the css

then in the elements using it `hidden` or `?hidden="${this.SomeBoolean}"`